wordpress-mobile/WordPress-iOS
# Reader: better voiceover support in cells

Implements the second part of #8667 

- I have maintained the button with the author name, blog title and published date as an independent control, because its action (browsing all the posts from that blog) is different than the action when tapping the cell (browsing the selected post in detail)
- I have still kept the `Accessible` interface around, but it seems unnecessary. I would still like to keep it around while I work on some other voiceover-related tickets, to decide if it actually helps or just adds unnecessary clutter.

![img_2544](https://user-images.githubusercontent.com/2722505/36569661-8155d7da-17e3-11e8-9a82-062b1383fe28.PNG)
![img_2545](https://user-images.githubusercontent.com/2722505/36569671-86716626-17e3-11e8-8048-245e32e80795.PNG)
![img_2546](https://user-images.githubusercontent.com/2722505/36569678-8b58a1ea-17e3-11e8-88de-363529bdb7ed.PNG)

To test:
- Navigate to Reader
- Activate Voiceover
- Tap a cell.

Expected behaviour: voiceover should be able to identify the following:
- Autor+blog title+ published date as a single control
- Post body
- Visit button
- Share button
- Like button
- Comments button
- More button
